src/views/WorkspaceDetail.vue

```diff
@@ -222,8 +220,7 @@ export default Vue.extend({
           this.editing = false;
           this.requestError = null;
 
-          // TODO: REMOVE THIS REF WHEN VUEX IS ADDED
-          this.$emit('update');
+          store.dispatch.fetchWorkspaces();
```

Comment:
Does this need an `await` plus possibly some other visual state logic (like the `this.loading` prop in the hunk below)? Could that be related to the bug @JackWilb mentioned?

Reply:
I don't think this is related to the bug @JackWilb found, as this is in the `renameWorkspace` method. Awaiting this should have no effect, as it's the last call in the method.